Web/starter_project_vanguard/tailwind.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
         danger: '#b91c1c',
         'primary-text': '#353535',
         'secondary-text': '#717171',
+        'navColor':'#565656'
```

Comment:
should we not have the color be a primary color ?
what part would be different in the navColor from the primary color?
as in what color in the nav, should only be in the nav?

Reply:
[nit] also let us stick to `kebab-case` as it is used with the others instead of `camelCase`

Reply:
the color on figma is not the same as what is defined primarily here..